observer.cpp

```diff
@@ -62,9 +63,14 @@ void observer::store_var_fit(const simulation& s)
     m_var_fitnesses.push_back(var_fitness(s));
 }
 
-void observer::store_top_n_inds(const simulation &s, int n)
+void observer::store_top_n_inds(const simulation &s)
 {
-    m_top_inds.push_back(get_best_n_inds(s, n));
+    m_top_inds.push_back(get_best_n_inds(s, m_top_proportion));
+}
+
+void observer::store_top_n_inds(const simulation &s, int proportion)
```

Comment:
should this (and the one above) empty the already stored individuals before pushing back?